QUESTIONS.md

```diff
@@ -1,5 +1,14 @@
 # Questions about Code, Node, Express, React
 
 What questions can you come up with?
-
+- More on React
+- More CSS please!
+- More HTML!
+- What is front end?
+ 
 ## Cohort Questions
+- When will profolio site be covered
+- Is there a syllabus we can see
+- Is there a way to know what assignments we still need to hand in
+- When will SQL be introduced // been told to use for industry project
+- When interviewed, I was vocal about how I think I am more design oriented and am curious about UX design. How can I apply what I have learned to that field? Will I have enough skills in that area for an entry level position? Will we learn more front end? What is front end?
```

Comment:
UX design is super cool, very involved. I know there are a few youtube channels specifically about it, though haven't dug in too deeply myself. From what I do know, the skills you've learned/are learning here are an awesome and broadly applicable palette that will help, but not quite enough to be a fully fledged UX dev. Brett or JR might have specific people you could meet, that's what I'd ask for!!